components/common/ModalPotal.tsx

```diff
@@ -0,0 +1,14 @@
+import { ReactNode } from 'react';
+import ReactDom from 'react-dom';
+
+interface Props {
+  children: ReactNode;
+}
+
+const ModalPortal = ({ children }: Props) => {
```

Comment:
```suggestion
const ModalPortal = ({ children }: { children: React.ReactNode }) => {
```

여기는 그냥 이케 써도 갠춘쓰~ children은 따로 인터페이스 안 만들기로 했으니께~